### DISCUSSION TITLE
Example projects using audio-tools

### DISCUSSION THREAD

**Discussion by kungfupizza** [pschatzmann/arduino-audio-tools#1822] [Q&A] [answered]
I am just wondering if there are any example projects using audio-tools library that I can refer when building my own home audio system.

Ideally a project making a audio-player with feature to play from SD card or https URLs.

If someone knows of one, I would like to refer to them.

**Comment by pschatzmann** [comment, marked as answer]
I suggest you try to understand the [naming conventions](https://github.com/pschatzmann/arduino-audio-tools/wiki/Examples) in the exaples.
